app/mailers/base_mailer.rb

```diff
@@ -3,15 +3,16 @@ class BaseMailer < ActionMailer::Base
 
   private
 
-  def set_recipients(recipients)
+  def set_recipients(recipients, cc = nil)
     # Sendgrid API allows a single SMTP request to send multiple
     # email messages. Change this to something else if we move
     # away from Sendgrid.
     # http://sendgrid.com/docs/API_Reference/SMTP_API/
 
     headers['X-SMTPAPI'] = {
-      to: recipients
+      to: recipients,
+      cc: cc
```

Comment:
I don't think Sendgrid actually accepts a `cc` field here, it isn't included in their docs and it didn't work when I tried it out on staging.

I think it probably makes more sense to just add the other organizers to the normal 'recipients'. The only value in showing them as a `cc` would be to reveal their emails to the other recipients, which is probably not necessary.